jest.config.ts

```diff
@@ -11,7 +11,7 @@ export default {
   // bail: 0,
 
   // The directory where Jest should store its cached dependency information
-  // cacheDirectory: "/private/var/folders/r4/b1879v110c79t2wv163m95300000gp/T/jest_dy",
+  // cacheDirectory: "/private/var/folders/cn/dnd_thyn0_zbdnz6dr8f7wdc0000gn/T/jest_dx",
```

Comment:
Could you move this change to the other PR?

I'd like the jest introduction to be somewhat compact, not containing settings back-and-forth that later no one will be able to understand.

... Feel free to rebase / squash / amend (the incoming commits) to your leisure.

Reply:
Just handle this change like `package-lock.json` and ignore it.